### PR TITLE
Base section offsets on header size

### PR DESCRIPTION
**Pull request by adborden**
- Remove padding on .main-content that throws off the wayfinder calculation
- Use offests based on header size for desktop

I haven't touched mobile yet. @cmc333333 pointed out in slack that this is still missing. We don't specify the mobile header size as a variable, and I'm reluctant to add more variables. If that's the best way, then so be it. Any other thoughts?

### CODE REVIEW

File: regulations/static/regulations/js/source/views/main/child-view.js [hunk 1]
```diff
@@ -82,7 +82,7 @@ var ChildView = Backbone.View.extend({
     // for the fixed position header, is deemed the active section
     checkActiveSection: function() {
       $.each(this.$sections, function(idx, $section) {
-        if ($section.offset().top + this.$contentHeader.height() >= $(window).scrollTop()) {
+        if ($section.offset().top >= $(window).scrollTop()) {
```
Comment (cmc333333) [inline regulations/static/regulations/js/source/views/main/child-view.js]: It looks like this is still needed:

Previous:
screen shot 2016-06-20 at 10 40 38 am


With Patch:
screen shot 2016-06-20 at 10 40 52 am

Comment (cmc333333) [inline regulations/static/regulations/js/source/views/main/child-view.js]: After an out-of-band chat, it looks like we want to add an offset of 30-50px (roughly one line of text) to keep previous functionality. We'll add a ticket around changing the logic to give a better user experience.

**Comment by adborden** [comment]
After talking with @cmc333333 offline, I added an offset of 64px (approx 2 * line-height + padding-top) so the wayfinder doesn't change until you scroll past the second line of the section. The offset is configurable by setting `window.WAYFINDER_SCROLL_OFFSET`, so in a pinch, other apps can configure this offset to match their section padding.

We discussed that the ideal case would be to only change the wayfinder once you've scrolled past the bottom of the section, but determining the bottom section isn't straight forward for nested sections. My idea was to change `this.$sections` to be a tree structure of sections and child-sections, then the search would become a depth-first traversal to find the current node. Seemed out of scope for this PR.

**Comment by cmc333333** [comment]
Thank you kindly! Works locally for ATF and N&C, too

**Pull request merged**